src/libraries/System.Private.Uri/tests/FunctionalTests/UriMailToTest.cs

```diff
@@ -29,10 +25,10 @@ public void UriMailTo_SchemeOnly_Success()
         public void UriMailTo_SchemeAndBackslash_Throws()
         {
             Assert.ThrowsAny<FormatException>(() =>
-          {
-              Uri uri = new Uri(@"mailto:\");
-              ValidateNotCorrupt(uri);
-          });
+            {
+                Uri uri = new Uri(@"mailto:\");
+                ValidateNotCorrupt(uri);
```

Comment:
This test is validating that the `new Uri(...)` throws, right?  What's the person of the line after it?  Isn't that dead code?  Same for cases below.

Reply:
Removed that helper method and changed call sites to expressions